samples/getting-started/DemoApp/ViewController.swift

```diff
@@ -19,8 +26,63 @@ class ViewController: UIViewController {
         label.font = .systemFont(ofSize: 30)
         label.sizeToFit()
         label.center = view.center
+        
+        buttonForAlert.setTitle("Show Alert", for: .normal)
+        buttonForAlert.titleLabel?.font = .systemFont(ofSize: 20)
+        buttonForAlert.sizeToFit()
+        buttonForAlert.center.x = view.frame.midX
+        buttonForAlert.center.y = label.frame.maxY + 30
+
+        buttonForActions.setTitle("Show Actions", for: .normal)
+        buttonForActions.titleLabel?.font = .systemFont(ofSize: 20)
+        buttonForActions.sizeToFit()
+        buttonForActions.center.x = view.frame.midX
+        buttonForActions.center.y = buttonForAlert.frame.maxY + 10
+        
+        #if os(iOS)
+        buttonForAlert.addTarget(self, action: #selector(objc_presentAlertController), for: .touchDown)
+        buttonForActions.addTarget(self, action: #selector(objc_presentActionsController), for: .touchDown)
```

Comment:
nit:
Normally we add the touch handlers on iOS for `.touchUpInside` - this allows the user to visually see that the button is pressed (as the color changes)